blocked-edges/4.18.0-rc.4-LabeledMachineConfigAndContainerRuntimeConfigBlocksMCO.yaml

```diff
@@ -0,0 +1,16 @@
+to: 4.18.0-rc.4
+# 4.17 before 4.17.12 OR 4.18.z with z<0-rc4
```

Comment:
declaring a risk for 4.18.0-rc.4 without declaring the risk for later 4.18.0-rc.* doesn't make sense to me, and we have a few of those later RCs:

```console
$ curl -s 'https://api.openshift.com/api/upgrades_info/graph?arch=multi&channel=candidate-4.18' | jq -r '.nodes[].version' | grep 0-rc | sort -V | tail -n7
4.18.0-rc.4
4.18.0-rc.5
4.18.0-rc.6
4.18.0-rc.7
4.18.0-rc.8
4.18.0-rc.9
4.18.0-rc.10
```

I think we should drop this entry, and rely on `4.18.0-rc.4-PreRelease.yaml` and similar `PreRelease` risks to keep folks from updating to those pre-releases, and save ourselves the effort of trying to figure out and declare which pre-releases are exposed to which risks, once a 4.y has gone GA and very few folks are expected to care about its pre-releases anymore.

Reply:
Removed.